Sources/BraintreeLocalPayment/BTLocalPaymentClient.swift

```diff
@@ -146,7 +146,9 @@ import BraintreeDataCollector
         requestParameters["_meta"] = metadataParameters
 
         apiClient.post("/v1/payment_methods/paypal_accounts", parameters: requestParameters) { [weak self] body, response, error in
-            guard let self else { return }
+            guard let self else {
+                return
```

Comment:
How do we want to handle the cases in this class? These methods don't have a completion and instead rely on `merchantCompletion` which relies on self, which we won't have access to. Should we log something to the console in this case?

Reply:
I'm down for a console log, but also don't feel strongly if it just no-ops. We could leave a TODO to refactor these methods to pass the completion like the other clients do

Reply:
Updated: https://github.com/braintree/braintree_ios/pull/1049/commits/46f409ef668c8c37343aad61b0903bddaf6fb067